README.md

```diff
@@ -111,8 +111,10 @@ The encode and protocol test suites does not require a
 running MySQL server on localhost.
 
 For the suites `mysql_tests`, `ssl_tests` and `transaction_tests` you need to
-start MySQL on localhost and give privileges to the user `otptest` additionally you need to set my.cnf with `local_infile=1`. And (for
-`ssl_tests`) to the user `otptestssl`:
+start MySQL on localhost and give privileges to the user `otptest`. 
+Additionally for `mysql_tests` the mysql server (my.cnf) needs to be configured 
+to allow file loading by adding the following the mysql configuration `local_infile=1`. 
+And (for `ssl_tests`) to the user `otptestssl`:
```

Comment:
It looks better, but I still don't like that this info is in the middle of how to setup test user privileges.

I think it would be better to put this about `local_infile=1` after the `GRANT` code block, next to the SSL files and config.

Is `local_infile=1` the default? The test passes in Travis without this, so it must be enabled by default in Travis at least.

If it's on by default, I think we should write something along the lines "Local infile needs to be enabled in the server, which it is by default. If you're having problems with the LOAD LOCAL INFILE test case, check that `local_infile=1` in my.cnf". But I don't know if it's on by default. What do you think?

Reply:
> Is `local_infile=1` the default? The test passes in Travis without this, so it must be enabled by default in Travis at least.

Try out gitkraken; it makes git a lot easier. 

No `local_infile=1` it is not default added it to the ssl configuration file; as far as test went on fedora even with sudo I couldn't append it to the mysql configure it is read and write only root; read group and user. I couldn't figure out a command to add it via the .travis file; I am not used to do either test cases for travis builds.

Reply:
Indeed, gitkraken looks like it gives a good overview of the branches and commits. I have only looked briefly at the website.

You can skip the travis file. The test case is passing in the builds so it seems to be enabled on Travis already.

Reply:
Hi! What's happening to this PR? This seems to be the only unresolved comment. Apart from that, everything seems to be done, or am I missing something?